src/other/Table.js

```diff
@@ -307,27 +325,40 @@
             .append("td")
         ;
         cells[this.renderHtmlDataCells() ? "html" : "text"](function (d, idx) { 
-            var retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            var retVal;
+            if (!(d instanceof Widget)) {
+                retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            }
             return retVal; 
```

Comment:
@GordonSmith it returns undefined here because the widget gets rendered in the cell here https://github.com/hpcc-systems/Visualization/pull/1237/files#diff-164179c53aa545afdc2035238236796aR351

Is it better to just return ""?  Does it matter?